src/Microsoft.DotNet.Interactive/FrontendEnvironment.cs

```diff
@@ -17,5 +19,10 @@ public virtual Task ExecuteClientScript(string code, KernelInvocationContext con
 
             return Task.CompletedTask;
         }
+
+        public virtual Task ForwardCommand(KernelCommand command, KernelInvocationContext context)
```

Comment:
Why not put this directly on the `JavaScriptKernel` (and make it private)?

Reply:
ok, so we pass the KernelClient object to the jskernel?

Reply:
The jskernel obtains a kernelClient for commadn forwarding, removed all additional logic from  FrontendEnvironment type